src/test/java/com/uid2/admin/vertx/SiteServiceTest.java

```diff
@@ -170,17 +175,15 @@ void addSiteExistingSites(Vertx vertx, VertxTestContext testContext) {
         Site[] initialSites = {
                 new Site(7, "initial_site", false),
         };
-        Site[] addedSites = {
-                new Site(8, "test_site", false),
-        };
+        Site addedSites = new Site(8, "test_site", false);
 
         setSites(initialSites);
 
         post(vertx, testContext, "api/site/add?name=test_site", "", response -> {
             assertAll(
                     "addSiteExistingSites",
                     () -> assertEquals(200, response.statusCode()),
-                    () -> checkSiteResponse(addedSites, new Object[]{response.bodyAsJsonObject()}));
+                    () -> checkSiteResponseWithoutCreatedAt(addedSites, response.bodyAsJsonObject()));
```

Comment:
I don't think checking createdAt for the newly created site is necessary. but if we want to check that we could consider passing clock into site service, which we could mock the clock in the tests